### PR TITLE
Pcc52

### PR DESCRIPTION
**Pull request by ttafsir**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [3]
Estimated time spent (hours): [1/4]
Completed (yes/no): [yes]
I stretched my coding skills (if yes what did you learn?): [it was fun]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 52/ttafsir/pomodoro.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                self._countdown += timedelta(seconds=-1)
+                sleep(1)
+                live.refresh()
+
+    def restart(self, start_timer, prompt):
+        self._countdown = timedelta(minutes=start_timer)
+        self.run(prompt)
+
+
+timer_duration = int(input("Duration: ").strip())
```
Comment (bbelderbos) [inline 52/ttafsir/pomodoro.py:29]: Put this into an `if __name__ == "__main__"` so it doesn't run if you would want to import this module.

File: 52/ttafsir/pomodoro.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from rich.console import Console
```
Comment (bbelderbos) [inline 52/ttafsir/pomodoro.py:1]: `rich`?

**Review by bbelderbos — APPROVED** [review]
NIce

File: 52/ttafsir/pomodoro.py (hunk 2 above)
Comment (ttafsir) [reply 52/ttafsir/pomodoro.py:1]: I've been playing with `rich` lately and it seems super cool and useful for CLI outputs and layouts.

File: 52/ttafsir/pomodoro.py (hunk 1 above)
Comment (ttafsir) [reply 52/ttafsir/pomodoro.py:29]: ah, good point. will do that.

Comment (ttafsir) [reply 52/ttafsir/pomodoro.py:29]: Thanks!

File: 52/ttafsir/pomodoro.py (hunk 2 above)
Comment (bbelderbos) [reply 52/ttafsir/pomodoro.py:1]: cool, need to check that out, maybe we can do an article on the blog ...

**Pull request merged**